recipes/libatomic_ops/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+project(test_package C)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
[cmake_minimum_required](https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html) must be the very first line of the file:
```
Call the cmake_minimum_required() command at the beginning of the top-level CMakeLists.txt file even before calling the project() command. It is important to establish version and policy settings before invoking other commands whose behavior they may affect. See also policy CMP0000.
```
otherwise, it may file (e.g. on Android we have seen such failures already)
for @uilianries: do we have hook check for that?

Reply:
Yes, https://github.com/conan-io/hooks/blob/9701cd214c293e46d668f2291254f40f30f3c593/hooks/conan-center.py#L265

I need to check why it didn't warned about :thinking:

Reply:
looking again, it only checks for `cmake_minimum_required` presence. It requires a new hook.